src/main/java/com/google/gradle/osdetector/OsDetector.java

```diff
@@ -26,18 +26,54 @@
 
   private static final Impl impl = new Impl();
 
-  static String os() {
+  public String getOs() {
     return (String) impl.detectedProperties.get(Detector.DETECTED_NAME);
   }
 
-  static String arch() {
+  public String getArch() {
     return (String) impl.detectedProperties.get(Detector.DETECTED_ARCH);
   }
 
-  static String classifier() {
+  public String getClassifier() {
     return (String) impl.detectedProperties.get(Detector.DETECTED_CLASSIFIER);
   }
 
+  public Release getRelease() {
+    Object releaseId = impl.detectedProperties.get(Detector.DETECTED_RELEASE);
+    if (releaseId == null) {
+      throw new UnsupportedOperationException("release is not available for " + getClassifier());
```

Comment:
I am OK with returning a null. `isReleaseSupported()` is probably unnecessary because it's idiomatic in Groovy to check a ``null`` return value. ``null`` is evaluated as ``false`` in conditional statement:
```groovy
if (osdetector.release) {
  osdetector.release.isLike(...)
}
```